task_list.py

```diff
@@ -0,0 +1,18 @@
+from tkinter import *
+
+#task_list.py houses the listbox settings for todo_list.py
+frame=Frame()
+task_list = Listbox(
+    frame,
+    width=22,
+    height=8,
+    bd=0,
+    foreground='#464646',
+    highlightthickness=0,
+    selectbackground = '#a6a6a6',
+    activestyle='none',
+    font=('Times', 18)
+)
+
+if __name__ == '__main__':
+    todo.mainloop()
```

Comment:
`todo` doesn't exist here, you would need to import it from `task_list` for this to be able to work

with that said, not every file needs an `if __name__ == '__main__'`, since this file exists just to define the `task_list` component, it's probably never going to be run on its own

if it is run on its own, what would you want it to do? how would it differ from running the main file? if it doesn't differ, there's no point in having two entry points, it's more code to maintain and more opportunity for bugs

Reply:
I don't think I want it to really do anything when run on it's own so I can remove that and simply have the task_list variable defined. But I also need the "frame" variable defined in the task_list.py file because otherwise I get an error complaining "frame" is not defined. This causes a second window to open when I run the main file and my listbox doesn't actually show up as intended.

Reply:
You might want to then wrap everything here into a function, and the function takes a `Frame` argument
and then you can call the function from the main file, passing in the frame

Reply:
I was once, but now I'm a professional couch potato